### ISSUE TITLE
Replace "caveat" with "warning"?

### ISSUE THREAD

**Issue by kaycebasques** [reactjs/reactjs.org#155]
Nice docs :)

Just a suggestion to replace "caveat", as seen here in [Introducing JSX](https://github.com/reactjs/reactjs.org/blob/master/content/docs/introducing-jsx.md)

```
>**Caveat:**
>
>Since JSX is closer to JavaScript than HTML, React DOM uses `camelCase` property naming convention instead of HTML attribute names.
>
>For example, `class` becomes [`className`](https://developer.mozilla.org/en-US/docs/Web/API/Element/className) in JSX, and `tabindex` becomes [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/tabIndex).
```

with a less-fancy / more straightforward word such as "warning"

**Comment by bvaughn** [comment]
Good suggestion. Would you be interested in submitting a PR for this?

**Comment by bvaughn** [comment]
Fixed via PR #163
